php/Terminus/Collections/SshKeys.php

```diff
@@ -50,9 +67,9 @@ public function deleteAll() {
    * @param array $options params to pass to url request
    * @return SshKeys $this
    */
-  public function fetch(array $options = array()) {
+  public function fetch(array $options = []) {
     $results = $this->getCollectionData($options);
-    $data    = $this->objectify($results['data']);
+    $data    = (object)$results['data'];
```

Comment:
Why am I turning this into an object and then iterating it as though it were an array? Keep this as an array.